xml/System/ArraySegment`1.xml

```diff
@@ -370,11 +370,14 @@
     </Member>
     <Member MemberName="CopyTo">
       <MemberSignature Language="C#" Value="public void CopyTo (T[] destination, int destinationIndex);" />
```

Comment:
shouldn't this have netcore-2.0 specific signatures for VB and C#?

Reply:
Same as the last two issues ... this is specific functionality of the formatters in this scenario. I can potentially see that we'd want to change this behavior, cause it seems weird, but at this stage it would have to be a separate enhancement IMO.